### ISSUE TITLE
node 14.18.2

### ISSUE THREAD

**Issue by jarrett-confrey** [sclorg/s2i-nodejs-container#315]
hello, i see that there is an rpm for node 14.18.2 available now. what is the process for generating new minor version images?

**Comment by pkubatrh** [comment]
The rpms first need to be released by Red Hat in RHSCL or RHEL8 (depending on what version are you looking for). Only after that happens we can create a new build of the images.

**Comment by jarrett-confrey** [comment]
im hoping to get the [ubi7/nodejs-14](https://catalog.redhat.com/software/containers/ubi7/nodejs-14/5f770e670531ff37c904327c) image updated. it does appear that a newer rpm is currently available (14.17.5 vs 14.18.2)
```
docker run -it registry.access.redhat.com/ubi7/nodejs-14:1-51 -- node --version
v14.17.5
```
```
docker run -it registry.access.redhat.com/rhscl/s2i-core-rhel7:1 /usr/bin/bash -c 'yum install -y -q rh-nodejs14 && /opt/rh/rh-nodejs14/root/usr/bin/node --version'
warning: /var/cache/yum/x86_64/7Server/ubi-server-rhscl-7-rpms/packages/rh-nodejs14-3.6-1.el7.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID fd431d51: NOKEY
Public key for rh-nodejs14-3.6-1.el7.x86_64.rpm is not installed
Importing GPG key 0xFD431D51:
 Userid     : "Red Hat, Inc. (release key 2) <security@redhat.com>"
 Fingerprint: 567e 347a d004 4ade 55ba 8a5f 199e 2f91 fd43 1d51
 Package    : redhat-release-server-7.9-6.el7_9.x86_64 (@anaconda/7.9)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
Importing GPG key 0x2FA658E0:
 Userid     : "Red Hat, Inc. (auxiliary key) <security@redhat.com>"
 Fingerprint: 43a6 e49c 4a38 f4be 9abf 2a53 4568 9c88 2fa6 58e0
 Package    : redhat-release-server-7.9-6.el7_9.x86_64 (@anaconda/7.9)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
v14.18.2
```

**Comment by pkubatrh** [comment]
Ah if the nodejs rpm update is already released that makes it easier to figure out when the official images get updated.
The 14.18.2 update was [released](https://access.redhat.com/errata/RHSA-2022:0041) on Jan 6, the regular [rebuild](https://access.redhat.com/errata/RHBA-2022:0107) of ubi7 images happened on Jan 11. The images' content is frozen a few days before the container images are shipped, so the nodejs update likely missed it by a day or two. These rebuilds happen approx. every 6 weeks, so you can expect the update to get into the official images sometime in February.
You can of course build your own image with the updated nodejs rpms using the sources from this repository before the official update if you need to have the image earlier.
